docs/develop/add-ins-with-angular2.md

```diff
@@ -10,13 +10,13 @@ localization_priority: Normal
 This article provides guidance for using Angular 2+ to create an Office Add-in as a single page application.
 
 > [!NOTE]
-> Do you have something to contribute based on your experience using Angular to create Office Add-ins? You can contribute to this article in [GitHub](https://github.com/OfficeDev/office-js-docs) or provide your feedback by submitting an [issue](https://github.com/OfficeDev/office-js-docs-pr/issues) in the repo. 
+> Do you have something to contribute based on your experience using Angular to create Office Add-ins? You can contribute to this article in [GitHub](https://github.com/OfficeDev/office-js-docs-pr) or provide your feedback by submitting an [issue](https://github.com/OfficeDev/office-js-docs-pr/issues) in the repo.
```

Comment:
Can you update the link to "this article" to actually link to this article instead of just the repo? This could potentially be an accessibility issue. Also, should you do that, the link should include [this article in GitHub]